data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml

```diff
@@ -0,0 +1,57 @@
+type: md
+title: Trajectories of the SARS-CoV-2 main protease protein from Folding@Home
+description: >
+  All-atom MD simulations of the SARS-CoV-2 main protease with small molecules, simulated using Folding@Home.
+  Each small molecule was run alone in solution and docked to Mpro to compute absolute free energies of binding.
+  Simulations were run using GROMACS-5.0.4 or GROMACS-2020 and are stored as compressed binary XTC files.
+  Absolute free energies were computed using an expanded ensemble scheme, using 40 lambdas to decouple the ligand interactions.
```

Comment:
Links to methodology, if possible? And to the input files?

Reply:
I have a link to methodology that we wrote up for our Moonshot release. Do you think this is sufficient? The input files are also accounted for and I'm transferring them to the bucket today.

Reply:
Yes, thanks!